util/chplenv/third_party_utils.py

```diff
@@ -13,10 +13,13 @@
 #
 @memoize
 def default_uniq_cfg_path():
+    chpl_home = os.environ.get('CHPL_HOME', '')
+    chpl_module_home = os.environ.get('CHPL_MODULE_HOME', '')
+    arch_get_lcd = (chpl_home == chpl_module_home) and (chpl_home != '')
```

Comment:
Now that this logic is being used in more than one place, I think it should be moved into a helper function in utils.py. Something like `is_using_chapel_module()`